relay-server/src/envelope.rs

```diff
@@ -568,6 +568,15 @@ pub struct ItemHeaders {
     #[serde(default, skip_serializing_if = "is_false")]
     spans_extracted: bool,
 
+    /// Whether or not logs have been extracted from the item.
+    ///
+    /// In order to only extract logs once from an item while through a
+    /// chain of Relays, a Relay that extracts logs from an item (typically
+    /// the first Relay) MUST set this flag to true so that upstream Relays do
+    /// not extract the logs again causing double counting of the logs.
```

Comment:
As David pointed out, wouldn't there be inherent duplication because multiple events might share the same trail of breadcrumbs?